services/dkp-insights-management/0.10.0/dkp-insights-management.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dkp-insights-management
+  namespace: ${releaseNamespace}
+spec:
+  chart:
```

```suggestion
spec:
  dependsOn:                                                                    
    - namespace: ${releaseNamespace}                                            
      name: kubefed
    - namespace: ${releaseNamespace}                                            
      name: traefik
  chart:
```
We depend on two CRDs: Middleware from Traefik and FederatedSecret from Kubefed.
This could spare us some crashlooping and failed attempts to install.